src/lib/still-camera.ts

```diff
@@ -40,6 +40,7 @@ export default class StillCamera {
       case 'BCM2711':
       case 'BCM2835 - Pi 3 Model B':
       case 'BCM2835 - Pi 3 Model B+':
+      case 'Raspberry Pi 3 Model B Plus Rev 1.3':
```

Comment:
Does `model` really come back with this string on a rev 1.3? Just seems odd that the naming convention would change just for a Pi 3 rev 1.3.

Reply:
Hello, yes it is the correct string, I just use the name throw by the error 
``Could not determine JPEG signature. Unknown system model '${systemInfo.model}'` `
and it works for me